Eat_and_Greet/src/comps/Textbox/textbox.tsx

```diff
@@ -1,20 +1,39 @@
 import React, { useState } from 'react';
-import { TextInput, View } from 'react-native';
+import { TextInput, View , StyleSheet} from 'react-native';
 import styles from './style';
 
 interface TextBoxProps {
-  placeholder: string;
+  placeholder ?: string;
+  top ?: any;
+  left ?: any;
+  bottom ?: any;
+  right ?: any;
+  width ?: any;
+  height ?: any;
 }
 
-const TextBox = ( {placeholder} : TextBoxProps ) => {
+const TextBox = ( {placeholder, top, left, bottom, right, width, height} : TextBoxProps ) => {
 const [text, setText] = useState('');
-
 const handleTextChange = (userInput: string) => {
     setText(userInput);
 };
+const localstyle = StyleSheet.create({
+  placement: { 
+      top: top,
+      bottom: bottom,
+      left: left,
+      right: right,
+      position: 'relative'
+  },
+  sizing: {
+      width: width,
+      height: height, 
+  }
+})
+
 
   return (
-    <View style={styles.container}>
+    <View style= {[styles.container, localstyle.placement, localstyle.sizing]}>
```

Comment:
I think this is styles.ScreenContainer now? Might be based off Rob's change. Remember to Rebase before pushing the final draft (I know his change isn't out yet so no sweat).

Reply:
Actually they seem separate, what is styles.container and is it different than ScreenContainer?